_includes/cloud/cloud-db-create-before-begin.md

```diff
@@ -0,0 +1,2 @@
+* [Create a database using the Cloud UI](/docs/cloud/cloud-databases/cloud-db-create), OR
+* [Create a database using the Cloud API](https://api-docs-featurebase-cloud.redoc.ly/v2#operation/createDatabase)
```

Comment:
Never use a version on these links. Always use /latest

Reply:
Oh! I didn't know that :D

Reply:
updated for all redocly links